### PR TITLE
bottom navigation bar update

### PR DESCRIPTION
**Pull request by elim322**
- add background active tint color for bottom navigation.
- move placeholder images for activity screen into placeholder folder in assets
- adjust sizing for bottom navigation bar icons and profile icon for header

### CODE REVIEW

File: js/screens/Activity/Activity.js [hunk 1]
```diff
@@ -1,12 +1,99 @@
...
+    <ImageBackground
+      source={require("../../assets/Background/background.png")}
+      style={styles.background}
+    >
+      <ScrollView>
+        <View style={styles.container}>
+          <View style={styles.pickActivity}>
+            <Text style={styles.title}>Pick an Activity</Text>
+          </View>
+          <View>
```
Comment (siddharthparmar7) [inline js/screens/Activity/Activity.js:17]: use Fragment.

File: js/screens/Activity/styles.js [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+import { StyleSheet, Dimensions } from "react-native";
+const styles = StyleSheet.create({
+  background: {
+    width: Dimensions.get("window").width,
+    height: Dimensions.get("window").height
+  },
+  container: {
+    flex: 1,
+    justifyContent: "space-evenly",
+    height: Dimensions.get("window").height * 1.02
```
Comment (siddharthparmar7) [inline js/screens/Activity/styles.js:10]: deconstruct height and width on the top so you can easily just type height and width without huge syntax.

File: js/screens/Activity/styles.js [hunk 3]
```diff
@@ -1,2 +1,35 @@
...
+  },
+  pickActivity: {
+    alignSelf: "center",
+    marginBottom: 30
```
Comment (siddharthparmar7) [inline js/screens/Activity/styles.js:14]: can you avoid this huge margin?

File: js/screens/Activity/ActivityContainer.js [hunk 4]
```diff
@@ -3,11 +3,13 @@ import Activity from "./Activity";
 import { View } from "react-native";
 
 export default class ActivityContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (siddharthparmar7) [inline js/screens/Activity/ActivityContainer.js:8]: if you are not declaring a state, you do not need a constructor.

File: js/screens/Activity/ActivityContainer.js [hunk 5]
```diff
@@ -3,11 +3,13 @@ import Activity from "./Activity";
 import { View } from "react-native";
```
Comment (siddharthparmar7) [inline js/screens/Activity/ActivityContainer.js:3]: remove unnecessary imports.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please do not forget to squash your commits. Make the requested changes. Thank you.

File: js/screens/Activity/styles.js (hunk 3 above)
Comment (elim322) [reply js/screens/Activity/styles.js:14]: the design requires it

**Pull request closed without merging**